kitsune/groups/migrations/0005_create_community_team.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations
+
+
+def create_community_team(apps, schema_editor):
```

Comment:
Generally speaking I am not against the migration necessarily but we can just create the group manually for now. Maybe what we should do in a different PR is review all the groups that support some kind of functionality and create them as in single migration and/or script. 

Feel free to keep it or drop it. Personally I would create it manually and I would handle the possiblityof not having the group available in the code.